Dangerfile

```diff
@@ -1,25 +1,26 @@
 # Sometimes its a README fix, or something like that - which isn't relevant for
 # including in a CHANGELOG for example
-declared_trivial = pr_title.include? "#trivial"
+
+has_app_changes = !files_modified.grep(/lib/).empty?
+has_test_changes = !files_modified.grep(/spec/).empty?
 
 if ["KrauseFx", "orta"].include?(pr_author) == false
-  warn("Author @#{pr_author} is not a contributor")
+  warn "Author @#{pr_author} is not a contributor"
 end
 
 if (pr_body + pr_title).include?("WIP")
-  warn("Pull Request is Work in Progress")
+  warn "Pull Request is Work in Progress"
```

Comment:
Not really, it gets confusing easily once you add a second parameter and / or an inline if

```ruby
warn "hi", 3 if something 5
```